fast64_internal/sm64/sm64_geolayout_writer.py

```diff
@@ -152,17 +154,18 @@ def getCameraObj(camera):
 		' is no longer in the scene.')
 
 def appendRevertToGeolayout(geolayoutGraph, fModel):
-	fModel.materialRevert = GfxList(fModel.name + "_" + 'material_revert_render_settings', 
+	fModel.materialRevert = GfxList(fModel.name + "_" + 'material_revert_render_settings',
 		GfxListTag.MaterialRevert, fModel.DLFormat)
-	revertMatAndEndDraw(fModel.materialRevert, 
+	revertMatAndEndDraw(fModel.materialRevert,
 		[DPSetEnvColor(0xFF, 0xFF, 0xFF, 0xFF),
 		DPSetAlphaCompare("G_AC_NONE")])
 
-	drawLayers = geolayoutGraph.getDrawLayers()
+	# Get all draw layers, turn layers into strings (some are ints), deduplicate using a set
+	drawLayers = set([str(layer) for layer in geolayoutGraph.getDrawLayers()])
```

Comment:
```suggestion
	drawLayers = set(str(layer) for layer in geolayoutGraph.getDrawLayers())
```

also is it certain that at this point `getDrawLayers` is defined? (I saw in other places that it can be undefined for some geonodes)
I don't need a detailed explanation, just yes or no is enough

Reply:
> also is it certain that at this point getDrawLayers is defined
yup!

also, great suggestion lol